vnext/PropertySheets/External/Microsoft.ReactNative.Uwp.CSharpModule.targets

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- 
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  Licensed under the MIT License.
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <ProjectReference Include="$(ReactNativeWindowsDir)\Microsoft.ReactNative\Microsoft.ReactNative.vcxproj">
+      <Project>{f7d32bd0-2749-483e-9a0d-1635ef7e3136}</Project>
+      <Name>Microsoft.ReactNative</Name>
+    </ProjectReference>
+    <ProjectReference Include="$(ReactNativeWindowsDir)\Microsoft.ReactNative.Managed\Microsoft.ReactNative.Managed.csproj">
+      <Project>{F2824844-CE15-4242-9420-308923CD76C3}</Project>
+      <Name>Microsoft.ReactNative.Managed</Name>
+    </ProjectReference>
+    <PackageReference Include="Microsoft.UI.Xaml">
```

Comment:
is this needed?

Reply:
The `Microsoft.ReactNative.Manage` reference, yes. That's where all of the native module reflection (and future code-gen) is.

The `Microsoft.UI.Xaml` reference, I'm not 100% sure. If you use xaml in your native module lib, and the app uses mux, is it okay if the lib doesn't use mux?

Reply:
yeah meant the MUX reference (I might need a "this is how to use GH for PRs" tutorial 😿).
I think this should be fine to remove. Not every module will need MUX (esp if they don't use any controls in MUX but just use the built-in WUX stuff)

Reply:
So how do we make sure they don't use a different version of mux than we do? We enforce it for the app right?

Reply:
We shouldn't require any MUX controls in the core. We will for WinUI3 but for everyone on system XAML, MUX should be not needed. Even if we were to have a MUX 2.x dependency from core, it _should_ be ok for the app to have its own version (they'll just be bloaty, hopefully realize that they're loading code "2x" and fix the import in their core or in the app)

Reply:
Fixed.